arangod/Agency/Job.h

```diff
@@ -263,8 +267,7 @@ inline arangodb::consensus::trans_ret_t generalTransaction(
 }
 
 inline arangodb::consensus::trans_ret_t transient(AgentInterface* _agent,
-                                                  Builder const& transaction,
-                                                  bool waitForCommit = true) {
```

Comment:
Consider setting the default for waitForSync in the header. Wouldn't you say that it helps readability?

Reply:
wait for commit was not used here